compute/instances/troubleshooting/bootscript-eol.mdx

```diff
@@ -0,0 +1,90 @@
+---
+meta:
+  title: Dealing with the end of life of the bootscript feature
+  description: This page helps you troubleshoot issues related to the EOL of the bootscript feature
+content:
+  h1: Dealing with the end of life of the bootscript feature
+  paragraph: This page helps you troubleshoot issues related to the EOL of the bootscript feature
+tags: bootscript boot-script instance boot script
+dates:
+  validation: 2023-02-20
+  posted: 2023-02-20
+categories:
+  - compute
+---
+
+Certain older legacy Instance types (e.g. **VC1-x**, **X64-x**, **Start1-x**, and **Stardust**) offered the default option to use a bootscript (a preconfigured boot configuration) to start your Instance. This option has become deprecated and will no longer be supported.
```

Comment:
DEV1, GP1 and STARDUST1 instance type also support bootscript as a user selectable option.

Reply:
```suggestion
Certain older legacy Instance types (e.g. **VC1-x**, **X64-x**, **Start1-x**, and **Stardust**) offered the default option to use a bootscript (a preconfigured boot configuration) to start your Instance. This boot method is also available as an option on **DEV1-x**, **GP1-x**, and **STARDUST1** Instances. The bootscript option has become deprecated and will no longer be supported.
```